src/Evaluate/Infrastructure/Command/Analyze/AnalyzeCommand.php

```diff
@@ -112,11 +122,17 @@ public function getViolationReport(): ?ViolationReportInterface
      */
     public function execute(ContextInterface $context): ContextInterface
     {
+        $projectConfigurationRequest = new ConfigurationRequestDto(
```

Comment:
As to me it should be on the Codebase level but not inside the command.

Reply:
Moved to Codebase level